.github/CONTRIBUTING.md

```diff
@@ -14,9 +14,13 @@ Please make sure to have the following prepared (where applicable)
 Please make sure an issue exists when submitting a PR that can be referenced and used to complete the PR template, as well as provided contextual information for reviewers.
 
 ## Making Changes
+To develop for the project, you'll need the do follow these steps:
+1. Have current [NodeJS LTS](https://nodejs.org) installed
```

Comment:
Kinda ambiguous, current varies all the time.  Maybe > 10 ? Or is that deliberate because we're constantly updating it?

Reply:
Good point about the >= 10 part.  I'll update this.